cli/tox.ini

```diff
@@ -13,6 +13,7 @@ deps =
   pytest-cov
   pytz
   teamcity-messages
+  retrying
```

Comment:
I'm really confused by this, so do we have 3 places to define dependencies for a package? Ideally we should have one place for dependencies and another one for dev / test dependencies. I've opened this ticket : https://jira.mesosphere.com/browse/DCOS_OSS-1762

But this should be tackled separately.